### ISSUE TITLE
Python type hints are a bit too bright/bold

### ISSUE THREAD

**Issue by thgeorgiou** [Monokai/monokai-pro-vscode#360]
Hello!

When using "type hints" in Python, I find that they get a too bright/bold color. For example, if you enable either `python.analysis.inlayHints.functionReturnTypes` or `python.analysis.inlayHints.variableTypes`, you get these bright boxes:

![image](https://github.com/Monokai/monokai-pro-vscode/assets/884404/4b6973ea-e180-4697-8fc6-0bdc58960295)

On other themes, it is usually something more subtle. For example, this is the same instance in Github Dark:

![image](https://github.com/Monokai/monokai-pro-vscode/assets/884404/bebe4cab-73d3-4968-bdc6-12e710a9743e)

Thanks for the great theme! :)

**Comment by C4Phoenix** [comment]
I agree that the default for this is currently very distracting. I found a workaround by manually overwriting the colors using the following settings in the settings.json to something like :
```json
"workbench.colorCustomizations": {
    "editorInlayHint.Background": "#403e41", 
    "editorInlayHint.foreground": "#f1f1f1", 
    "editorInlayHint.typeBackground": "#403e41",
    // "editorInlayHint.typeForeground": "#78dce8",
    "editorInlayHint.typeForeground": "#5dabb4",
    "editorInlayHint.parameterBackground": "#403e41",
    "editorInlayHint.parameterForeground": "#fc9855",
},
```
